tests/src/test/scala/cats/tests/FunctionTests.scala

```diff
@@ -45,7 +44,6 @@ class FunctionTests extends CatsSuite {
   checkAll("Function0[Eqed]", EqLawTests[Function0[Eqed]].eqv)
   checkAll("Function0[POrd]", PartialOrderLawTests[Function0[POrd]].partialOrder)
   checkAll("Function0[Ord]", OrderLawTests[Function0[Ord]].order)
-  checkAll("Function0[Hsh]", HashLaws[Function0[Hsh]].hash)
```

Comment:
Hmm, was this test removed intentionally?

Reply:
Good catch, I removed it initially for erroring for erroring out, as with the new setup, all the laws are tested and you can't single out one specific law to run. I wanted to replace it with a manual test, but forgot, will fix :)